Recruitment/Exercises/devops_engineer.md

```diff
@@ -0,0 +1,25 @@
+# DevOps Engineer - Technical Test
+We think infrastructure is best represented as code, and provisioning of resources should be automated as much as possible.
+
+Your task is to create a CI build pipeline that deploys a web application to a load-balanced
+environment. You are free to complete the test in a local environment (using tools like Vagrant and
+Docker) or use any CI service, provisioning tool and cloud environment you feel comfortable with (we
+recommend creating a free tier account so you don't incur any costs).
```

Comment:
I'm not sure I'd recommend someone to waste their free tier for a home test like this one. Otherwise sounds good to me.